service/secret_service.go

```diff
@@ -81,3 +82,17 @@ func (s secretService) List(ctx context.Context, projectName string) ([]models.S
 	}
 	return secretItems, nil
 }
+
+func (s secretService) Delete(ctx context.Context, prjName string, secretName string) error {
```

Comment:
should consider the namespace name in the request and should use just project name for deleting secrets at project level & namespace name must be used for deleting secrets at namespace level, an extra bit of validation is always safe.

Reply:
Sure, will add extra validation